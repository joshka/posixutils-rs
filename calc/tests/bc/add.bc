1 + 2
1 + 2.00
1.00 + 2
45.89 + 2.3
quit
