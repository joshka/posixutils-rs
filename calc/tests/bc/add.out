3
3.00
3.00
48.19
